sdk/core/azure-core-test/src/main/java/com/azure/core/test/TestBase.java

```diff
@@ -193,7 +191,14 @@ public static boolean shouldClientBeTested(HttpClient client) {
                 .contains(configuredHttpClient.trim().toLowerCase(Locale.ROOT)));
     }
 
-    private static TestMode initializeTestMode() {
+    /**
+     * Initializes the {@link TestMode} from the environment configuration {@code AZURE_TEST_MODE}.
+     * <p>
+     * If {@code AZURE_TEST_MODE} isn't configured or is invalid then {@link TestMode#PLAYBACK} is returned.
+     *
+     * @return The {@link TestMode} being used for testing.
+     */
+    public static TestMode initializeTestMode() {
```

Comment:
I saw this logic was being used in multiple places so I exposed this as a utility method

Reply:
Reverted as some sub-classes override this (which should be removed)